features/dd-sdk-android-logs/src/main/kotlin/com/datadog/android/log/Logs.kt

```diff
@@ -51,4 +52,63 @@ object Logs {
     fun isEnabled(sdkCore: SdkCore = Datadog.getInstance()): Boolean {
         return (sdkCore as FeatureSdkCore).getFeature(Feature.LOGS_FEATURE_NAME) != null
     }
+
+    /**
+     * Add a custom attribute to all future logs sent by loggers created from the given SDK core.
+     *
+     * Values can be nested up to 10 levels deep. Keys
+     * using more than 10 levels will be sanitized by SDK.
+     *
+     * @param key the key for this attribute
+     * @param value the attribute value
+     * @param sdkCore the [SdkCore] instance to add the attribute to. If not provided, the default
+     * instance is used.
+     */
+    @JvmOverloads
+    @JvmStatic
+    fun addAttribute(key: String, value: Any?, sdkCore: SdkCore = Datadog.getInstance()) {
```

Comment:
should we accept `Any?` or just `Any` ?

Reply:
I kept it in line with Logger's interface which accepts `Any?`